File: iosApp/iosApp/Statistics/Statistics/StatisticsNavigationEvent.swift
enum StatisticsNavigationEvent: String, CaseIterable, Identifiable {
	case showBestTurn
	case showBiggestFinalTurn
	case showAverageValues
	case showShotDistribution
	case showVictoryDistribution
	case showSectorHeatmapDistribution
	case showTimeStatistics
	
	var id: String {
		self.rawValue
	}
}

File: androidApp/src/main/java/com/alextos/darts/android/statistics/statistics/StatisticsNavigationEvent.kt
package com.alextos.darts.android.statistics.statistics

sealed class StatisticsNavigationEvent {
    object ShowBestTurn: StatisticsNavigationEvent()
    object ShowBiggestFinalTurn: StatisticsNavigationEvent()
    object ShowAverageValues: StatisticsNavigationEvent()
    object ShowShotDistribution: StatisticsNavigationEvent()
    object ShowVictoryDistribution: StatisticsNavigationEvent()
    object ShowSectorHeatmapDistribution: StatisticsNavigationEvent()
    object ShowTimeStatistics: StatisticsNavigationEvent()
}